src/main/java/mods/eln/Eln.java

```diff
@@ -2682,6 +2695,15 @@ void registerHeatFurnace(int id) {
 			transparentNodeItem.addDescriptor(subId + (id << 6), desc);
 		}
 
+		{
+			subId = 1;
+			name = TR_NAME(Type.NONE, "Fuel Heat Furnace");
+
+			FuelHeatFurnaceDescriptor desc = new FuelHeatFurnaceDescriptor(name,
+				obj.getObj("FuelHeater"), new ThermalLoadInitializerByPowerDrop(780, -100, 10, 2));
```

Comment:
Do you know how the ThermalLoadInitializerByPowerDrop works? If so, I'd adore if you could add a bit of documentation.